sites/public/src/components/shared/FormSummaryDetails.tsx

```diff
@@ -177,8 +177,10 @@ const FormSummaryDetails = ({
 
   const allListingUnitTypes = getUniqueUnitTypes(listing?.units)
 
-  const preferredUnits = application.preferredUnit?.map((unit) => {
-    const unitDetails = allListingUnitTypes?.find((unitType) => unitType.id === unit.id)
+  const preferredUnits = application.preferredUnitTypes?.map((unit) => {
```

Comment:
We actually weren't showing preferred units on the summary page because of a typo here that would have been highlighted if we hadn't typed the application as `any` above (the data was still coming through correctly)